pkg/docker/create_devcontainer.go

```diff
@@ -40,7 +40,7 @@ type DevcontainerPaths struct {
 }
 
 type CreateDevcontainerOptions struct {
-	ProjectDir string
+	WorkspaceDir string
 	// Name of the project inside the devcontainer
 	ProjectName       string
```

Comment:
Should be `WorkspaceName`

Also, `ProjectTarget` should be `WorkspaceDirTarget`